xml/System/Console.xml

```diff
@@ -190,7 +190,7 @@
 
  If your app depends on specific code page encodings, you can still make them available by doing the following *before* you call any <xref:System.Console> methods:
 
-1.  Add a reference to the System.Text.Encoding.CodePages.dll assembly to your project.
+1.  Add a reference to the [System.Text.Encoding.CodePages package](https://www.nuget.org/packages/System.Text.Encoding.CodePages) to your project.
```

Comment:
>Add a reference to the [System.Text.Encoding.CodePages package](https://www.nuget.org/packages/System.Text.Encoding.CodePages) to your project. [](http://example.com/codeflow?start=4&length=143)

This package became inbox in .NET 6.0. may be adding a small sentence clarify that. like "This step is not needed if running on .NET 6.0 or later). Or we just delete this step as we don't support pre .NET 6.0 releases anymore?

Reply:
I have deleted it.